tests/Backend/Snowflake/TableDefinitionOperationsTest.php

```diff
@@ -77,6 +79,712 @@ public function testResponseDefinition(): void
+            'CHARACTER' => [
+                'working' => [
+                    'value' => '\'T\'',
+                    'expectFail' => [],
+                ],
+                'empty' => [
+                    'value' => '',
+                    'expectFail' => [],
+                ],
+                'not quoted' => [
+                    'value' => 'test',
+                    'expectFail' => [],
+                ],
+                'wrong quoted' => [
+                    'value' => '"test"',
+                    'expectFail' => [],
+                ],
+                'type number' => [
+                    'value' => 1,
+                    'expectFail' => [],
+                ],
+            ],
+            'STRING' => [
+                'working' => [
+                    'value' => '\'T\'',
+                    'expectFail' => [],
+                ],
+                'empty' => [
+                    'value' => '',
+                    'expectFail' => [],
+                ],
+                'not quoted' => [
+                    'value' => 'test',
+                    'expectFail' => [],
+                ],
+                'wrong quoted' => [
+                    'value' => '"test"',
+                    'expectFail' => [],
+                ],
+                'type number' => [
+                    'value' => 1,
+                    'expectFail' => [],
+                ],
+            ],
+            'TEXT' => [
+                'working' => [
+                    'value' => '\'T\'',
+                    'expectFail' => [],
+                ],
+                'empty' => [
+                    'value' => '',
+                    'expectFail' => [],
+                ],
+                'not quoted' => [
+                    'value' => 'test',
+                    'expectFail' => [],
+                ],
+                'wrong quoted' => [
+                    'value' => '"test"',
+                    'expectFail' => [],
+                ],
+                'type number' => [
+                    'value' => 1,
+                    'expectFail' => [],
+                ],
+            ],
+            'BOOLEAN' => [
+                'working bool bool string' => [
+                    'value' => 'true',
+                    'expectFail' => [],
+                ],
+                'working bool false string' => [
+                    'value' => 'false',
+                    'expectFail' => [],
+                ],
+                'working bool true' => [
+                    'value' => true,
+                    'expectFail' => [],
+                ],
+                'working bool false' => [
+                    'value' => false,
+                    'expectFail' => [],
+                ],
+                'empty' => [
+                    'value' => '',
+                    'expectFail' => [],
+                ],
+                'fail type' => [
+                    'value' => 'test', // SNFLK will take is false when escaped
```

Comment:
```suggestion
                    'value' => 'test', // SNFLK will take as false when escaped
```

Reply:
tvl fakt! 
![image](https://github.com/keboola/storage-api-php-client/assets/642928/360eb3cd-afee-4f9b-b8f0-c50df97eee7f)

Reply:
Akorát to pak spadne ve SNFLK na insertu 
![image](https://github.com/keboola/storage-api-php-client/assets/642928/c085f44d-154a-4a19-9e7c-586228e5e1bf)

Reply:
```
CREATE OR REPLACE TABLE mytable (id boolean DEFAULT 'test', num number);
SHOW SCHEMAS;
USE SCHEMA WORKSPACE_947738;
DESC TABLE mytable;
INSERT INTO mytable (num) VALUES (1);

```

Reply:
Tj. bude nám to sypat app errory při importu

Reply:
:(

Reply:
vyřešíme v separatní issue pro import